pandas/_libs/hashtable_func_helper.pxi.in

```diff
@@ -241,7 +241,7 @@ def ismember_{{dtype}}({{scalar}}[:] arr, {{scalar}}[:] values):
 
     # construct the table
     n = len(values)
-    kh_resize_{{ttype}}(table, min(n, len(values)))
+    kh_resize_{{ttype}}(table, n)
```

Comment:
dead code, and warning about min comparing signed and unsigned. fixing one fixes the other.

Reply:
Makes sense